### ISSUE TITLE
VV - Newsletter plugin

### ISSUE THREAD

**Issue by pratsajency** [ajency/Thisandthat#22]
1. Add the below plugin 
   http://www.verticalresponse.com/wordpress-newsletter-plugin
2. This Form will have only "Email Id" as input from the Users
3. Link this to 
   a. The "Sign-up for our Newsletter" under "STAY CONNECTED" section near the footer
   b. Also on http://www.videovolunteers.org/take-action/interact-with-us/, there is an "Sign up for our quarterly newsletter" option.

**Comment by jeromie** [comment]
@pratsajency any reason for this particular plugin? I created an account on VerticalResponse, but after login, there is no link to get the wordpress plugin. I did a search in wordpress plugins for the same and got two results. 
1. https://wordpress.org/plugins/vertical-response-newsletter-widget/ (this is an outdated plugin so havent tried it)
2. https://wordpress.org/plugins/verticalresponse-form/

I installed the second one on VV, but the login credentials of VerticalResponse are not being accepted so I cant proceed with using it.
u: anuj@ajency.in
p: Ajency@123

Can we use some other plugin for this? Or do they have a VerticalResponse account for newsletters which is why we need this plugin?

**Comment by anujkhurana** [comment]
Yes. VV gets a not for profit discount from VR (which is a salesforce
company). So we need to stick to VR.

**Comment by jeromie** [comment]
new plugin added which gives us a script to place anywhere on the site for a form to collect name and email id. Once submitted, details are stored in VR list of contacts.
